### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getSelections' of undefined

### ISSUE THREAD

**Issue by jphatton** [blakeembrey/atom-alignment#10]
This gets thrown when you try to instantiate without an open or selected window.

**Atom Version**: 0.176.0
**System**: Mac OS X 10.10.2
**Thrown From**: [alignment](https://github.com/blakeembrey/atom-alignment) package, v1.1.3

### Stack Trace

Uncaught TypeError: Cannot read property 'getSelections' of undefined

```
At /Users/jory/.atom/packages/alignment/lib/atom-alignment.coffee:21

TypeError: Cannot read property 'getSelections' of undefined
  at Object.module.exports.align (/Users/jory/.atom/packages/alignment/lib/atom-alignment.coffee:21:11)
  at atom-workspace.<anonymous> (/Users/jory/.atom/packages/alignment/lib/atom-alignment.coffee:17:14)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app/node_modules/command-palette/lib/command-palette-view.js:120:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app/node_modules/archive-view/node_modules/atom-space-pen-views/lib/select-list-view.js:335:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app/node_modules/archive-view/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:170:20)

```

### Commands

```
     -0:05.1 command-palette:toggle (atom-pane.pane.active)
     -0:00.8 core:confirm (atom-text-editor.editor.mini)
     -0:00.7 alignment (atom-pane.pane.active)
```

### Config

```json
{
  "core": {
    "excludeVcsIgnoredPaths": false,
    "disabledPackages": [
      "autoclose-html",
      "inline-autocomplete",
      "file-icons"
    ],
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  },
  "alignment": {}
}
```

### Installed Packages

```coffee
# User
Atom-Syntax-highlighting-for-Sass, v0.5.0
alignment, v1.1.3
angularjs, v0.1.0
autocomplete-plus, v1.1.0
color-picker, v1.4.4
javascript-snippets, v1.0.0
language-diff, v0.2.0
language-rspec, v0.2.1
monokai, v0.11.0
rspec-snippets, v0.4.0

# Dev
No dev packages
```

/cc @atom/core

**Comment by blakeembrey** [comment]
Thanks for the heads up! :+1:

**Comment by blakeembrey** [comment]
Patched with `v1.1.4`
